tools/toktx/toktx.cc

```diff
@@ -1237,9 +1241,8 @@ toktxApp::main(int argc, _TCHAR *argv[])
                     goto cleanup;
                 }
 
-                // TODO: add an option for renormalize;
-                //if (options.gmopts.mipRenormalize)
-                //    levelImage->renormalize_normal_map();
+                if (options.normalize)
```

Comment:
Where is normalization of the base image?

Reply:
line 906. Is that not the right place?